reference/UiTPAS.v2.json

```diff
@@ -3053,6 +3833,212 @@
         "x-tags": [
           "Models"
         ]
+      },
+      "Reward": {
+        "type": "object",
+        "title": "Reward",
+        "description": "",
+        "x-tags": [
+          "Models"
+        ],
+        "properties": {
```

Comment:
These are a lot of properties, I think it would be good to add a description to each one (or at least most) to explain what purpose they serve. Not every one is self-explanatory imho.

Reply:
\+ Sorting them so the required ones are shown first might make it easier to see at a glance which ones are always needed and might make the list less overwhelming.

Reply:
@erwin1 I see most have a description now, but I think you overlooked a couple. Can you also add some for `moreInfoUrl`, `cashInType`, `owningCardSystem`? Thanks!

Also some general feedback on the descriptions: A couple now describe the property from a very technical or abstract point of view, or basically copy the property name, like for example:

- _Period entity with a begin and end date._ (for `publicationPeriod`)
- _Comma separated list of labels._ (for `labels`)
- _Promotion description_ (for `promotionalDescription`)
 
While techically correct, I feel like these description do not add that much info that could not be interpreted from the schema already.

Other descriptions are much better though:

- _List of CardSystems whose members can exchange this Reward._ (for `applicableCardSystems`) => **describes functionality, i.e. what does this property do**
- _List of types of this Reward. This is free text, but common types are: "Gadget of item", "Doen", "Eten en drinken" and "Goede doel"._ (for `types`) => **describes common use cases**
- _Only for Reward of cashInType TICKETSALE. Describes the discount a passholder can receive in exhange of this Reward._ (for `ticketSaleDiscount`) => **describes constraints**

These examples contain information that cannot be conveyed from the property name or other info that is in the technical schema.

This being said, the descriptions don't have to be perfect right now. It's something we can always change, and also a work in progress for the UiTdatabank APIs. I think at some point we'll have to let someone proof-read everything and focus on the readability of the docs, aside from how the schema is structured etc. I'll discuss that further internally first. For now, I just wanted to point out that the second kind of descriptions that you provided are super useful and the more we can have of those the better! 😉